allensdk/brain_observatory/ecephys/stimulus_analysis/__main__.py

```diff
@@ -29,19 +32,23 @@ def calculate_stimulus_metrics(args):
                  StaticGratings,
                  NaturalScenes,
                  #NaturalMovies,
-                 #DotMotion,
+                 DotMotion,
                  Flashes,
                  ReceptiveFieldMapping,
                 )
 
-    df = reduce(lambda output, nwb_path: \
-                 pd.concat((output,
-                           add_metrics_to_units_table(nwb_path, stimulus_classes, args))),
-                 args['nwb_paths'],
-                 pd.DataFrame()
-                 )
+    result = Parallel(n_jobs=16, verbose=10) \
+            (delayed(add_metrics_to_units_table)(path, stimulus_classes, args) 
+                for path in args['nwb_paths'])
+
+    #df = reduce(lambda output, nwb_path: \
```

Comment:
general comment: there is a bunch of commented code scattered around. What's the deal? Are these TODO features, alternate implementations? Can we delete them?

Reply:
You need to make set these to \todo if you want doxygen to see it